### PR TITLE
Add expand arrow for mobile checkout order summary

### PR DESCRIPTION
**Pull request by ScottPolhemus**
Adds the expand arrow for the updated order summary designs here: https://casper.box.com/s/ycs0qxiijbovwjvjzseic4xgig4vyabb

(Yes, it is slightly different from the existing down arrow.)

### CODE REVIEW

File: lib/storefront/arrow-expand.svg [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 13 8">
+  <title>
+    Expand Arrow
+  </title>
+  <g>
```
Comment (victorwhy) [inline lib/storefront/arrow-expand.svg]: can you add a `desc` tag as well?

Comment (ScottPolhemus) [reply lib/storefront/arrow-expand.svg]: sure

**Comment by victorwhy** [comment]
will this replace the other arrow? if so, can we take the other one out?

**Comment by ScottPolhemus** [comment]
No, the other arrow is still used in several other places on the site and this was provided specifically for the order summary.

File: lib/storefront/arrow-expand.svg (hunk 1 above)
Comment (ScottPolhemus) [reply lib/storefront/arrow-expand.svg]: done

**Pull request merged**